<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>borrowReturn</class>
 <widget class="QDialog" name="borrowReturn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="borrowButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrow Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Return Book</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>411</width>
     <height>241</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>4</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Borrowed On</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Borrowed By</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>001</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>The Great Gatsby</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>002</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>1984</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>003</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>Brief Answers to the Big Questions</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>11/20/2024</string>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string>Bob</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>004</string>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string>Alexander Hamilton</string>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item row="3" column="3">
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
